website_user_login_redirect_custom/tests/test_website_login_redirect.py

```
@@ -0,0 +1,118 @@
+        )
+        self.assertEqual(enabled, "0")
+
+    def test_enable_redirect_feature(self):
+        """Test enabling redirect feature through settings"""
+        settings = self.env["res.config.settings"].create({})
+        settings.website_login_redirect_enabled = True
+        settings.website_login_redirect_url = "/shop"
+        self.env.flush_all()
+        settings.execute()
+
+        self.assertEqual(
```

Ruff (0.14.0)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)